src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/PlacementInfoExtensions.cs

```diff
@@ -64,7 +55,12 @@ public static AlternatesCollection Combine(this AlternatesCollection first, Alte
             }
             else if (second != null)
             {
-                first.AddRange(second);
+                AlternatesCollection combined = new();
```

Comment:
var combined = new AlternatesCollection();

Reply:
Updated call to constructor.